src/main/java/dev/aziz/movies/controllers/MovieController.java

```diff
@@ -27,4 +34,19 @@ public ResponseEntity<List<ReducedMovieDto>> getAllMovies() {
     public ResponseEntity<MovieDto> getMovie(@PathVariable Long id) {
         return new ResponseEntity<>(movieService.findMovieById(id), HttpStatus.OK);
     }
+
+    @PostMapping("/movies")
+    public ResponseEntity<MovieDto> createMovie(@RequestBody @Valid MovieDto movieDto) {
+        return new ResponseEntity<>(movieService.createMovie(movieDto), HttpStatus.CREATED);
```

Comment:
When creating an entity, it's a best practice return the URL where to find the created entity. You can easily achieve this using `ResponseEntity.create()`